string node_id       # The ID of the node providing the methods.
string[] methods     # The list of available methods.
---
